open-api.yaml

```diff
@@ -748,6 +748,16 @@ components:
 
         > warn
         > Attribute(s) used in `filter` should be declared as filterable attributes. See [Filtering and Faceted Search](https://docs.meilisearch.com/reference/features/filtering_and_faceted_search.html).
+    fields:
+      name: fields
+      in: query
+      description: 'Comma-separated list of fields to display for an API resource. By default it contains all fields of an API resource.'
+      schema:
+        type: string
+        items:
+          type: string
+        example: 'uid,createdAt'
```

Comment:
Just checking:

`?fields=` => 200 OK, full document object.
`?fields=wrongName` => 200 OK, full document object.

This is correct?

Reply:
Good one @brunoocasali; I think it should be equivalent to the current attributesToRetrieve behavior.

- Not sending the `fields` parameter = `fields=*`
- Sending `fields=` returns empty documents
- Sending `fields=wrongField` returns empty documents
- Sending `fields=wrongField,fieldThatExists` return documents with fieldThatExists displayed

WDYT?

Reply:
Perfect for me!